deploy/olm-catalog/mobile-developer-console-operator/0.2.6/mdc_v1alpha1_mobiledeveloperconsole_crd.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: mobileclients.mdc.aerogear.org
+spec:
+  group: mdc.aerogear.org
+  version: v1alpha1
+  scope: Namespaced
+  names:
+    plural: mobileclients
+    singular: mobileclient
+    kind: MobileClient
```

Comment:
This is still the wrong CRD like before.

If you first create a file called `deploy/olm-catalog/csv-config.yaml`, with the contents:
```
crd-cr-paths:
  - deploy/crds
  - deploy/mdc_v1alpha1_mobileclient_crd.yaml
```

then it should create the correct one.

This `MobileClients` CR is an odd one, because it's not really managed by the operator (it's managed by MDC currently). Maybe for now we should copy it into an equivalent file as this one, and also maybe add it to the list of `owned` CRDs in the CSV spec? I'm not sure though, and would like to get other opinions. @philbrookes @maleck13 @wei-lee any thoughts?

Maybe adding the MobileClient CRD as an api here in the operator might get it to generate correctly? Not sure of any side effects of that, so let's not worry about that in this PR.

Reply:
So MDC itself does things with the CR and the operator doesn't do anything? Hmmm. Well I still think it makes sense to be in the operator CSV and probably owned as this way OLM will ensure it gets installed when the operator is installed.

Reply:
@maleck13 yes, MDC is like an operator that runs in the browser as the user, rather than as a ServiceAccount.